reoptjl/models.py

```diff
@@ -1728,17 +1784,9 @@ class ElectricUtilityInputs(BaseModel, models.Model):
         default=list, blank=True,
         help_text=("PM2.5 emissions factor over all hours in one year. Can be provided as either a single constant fraction that will be applied across all timesteps, or an annual timeseries array at an hourly (8,760 samples), 30 minute (17,520 samples), or 15 minute (35,040 samples) resolution.")
     )
-    emissions_factor_CO2_decrease_fraction = models.FloatField(
-        default=0.01174,
-        validators=[
-            MinValueValidator(-1),
-            MaxValueValidator(1)
-        ],
-        null=True, blank=True,
-        help_text="Annual percent decrease in the total annual CO2 marginal emissions rate of the grid. A negative value indicates an annual increase."
-    )
+    
     emissions_factor_NOx_decrease_fraction = models.FloatField(
-        default=0.01174,
+        default=0.02163,
```

Comment:
since we now have these defaults in a dictionary object REopt.jl, perhaps we add something similar to `FUEL_DEFAULTS` here, and then we can populate the default values as needed in `clean()` for this and the other pollutants?

Reply:
added in https://github.com/NREL/REopt_API/pull/475/commits/5c603ca03226e43a4791992bf10f7cc6ad120936 but I should add a test to make sure it's working